test/TestProjects/BodyAndPath-LowLevel/Generated/BodyAndPathClient.cs

```diff
@@ -55,11 +55,12 @@ public BodyAndPathClient(AzureKeyCredential credential, Uri endpoint = null, Bod
         /// <param name="content"> The content to send as the body of the request. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="itemName"/> or <paramref name="content"/> is null. </exception>
+        /// <exception cref="ArgumentException"> <paramref name="itemName"/> is an empty string, and was expected to be non-empty. </exception>
 #pragma warning disable AZC0002
         public virtual async Task<Response> CreateAsync(string itemName, RequestContent content, RequestContext context = null)
 #pragma warning restore AZC0002
         {
-            Argument.AssertNotNull(itemName, nameof(itemName));
+            Argument.AssertNotNullOrEmpty(itemName, nameof(itemName));
```

Comment:
Just to make sure I'm understanding correctly: itemName is a path parameter here?

Reply:
Yes.